app/ocr.py

```diff
@@ -117,54 +118,8 @@ class BIACase:
     with open('./app/court_locations.json') as f:
         court_locs = json.load(f)
 
-    appellate_panel_members = [
-        "Adkins-Blanch, Charles K.",
-        "Michael P. Baird",
-        "Cassidy, William A.",
-        "Cole, Patricia A.",
-        "Couch, V. Stuart",
-        "Creppy, Michael J.",
-        "Crossett, John P.",
-        "Donovan, Teresa L.",
-        "Foote, Megan E.",
-        "Geller, Joan B.",
-        "Gemoets, Marcos",
-        "Gonzalez, Gabriel",
-        "Goodwin, Deborah K.",
-        "Gorman, Stephanie E.",
-        "Grant, Edward R.",
-        "Greer, Anne J.",
-        "Guendelsberger, John",
-        "Hunsucker, Keith E.",
-        "Kelly, Edward F.",
-        "Kendall Clark, Molly",
-        "Liebmann, Beth S.",
-        "Liebowitz, Ellen C.",
-        "Mahtabfar, Sunita B.",
-        "Malphrus, Garry D.",
-        "Mann, Ana",
-        "Miller, Neil P.",
-        "Monsky, Megan Foote",
-        "Montante Jr., Phillip J.",
-        "Morris, Daniel",
-        "Mullane, Hugh G.",
-        "Neal, David L.",
-        "Noferi, Mark",
-        "O'Connor, Blair",
-        "O'Herron, Margaret M.",
-        "O'Leary, Brian M.",
-        "Owen, Sirce E.",
-        "Pauley, Roger",
-        "Petty, Aaron R.",
-        "Pepper, S. Kathleen",
-        "RILEY, KEVIN W.",

```

Comment:
Should "RILEY, KEVIN W." be formatted this way? I could see some potential issues with querying down the road.

Reply:
🤔 I agree with that as well, Andrew.

Reply:
```
    with open('./app/judge_names.pkl', 'rb') as j:
        appellate_panel_members = pickle.load(j)
```

`I` believe all of those lines were deleted and replaced by the loaded pickle file.

Reply:
Great👍🏼